include/deal.II/base/function_signed_distance.h

```diff
@@ -254,6 +260,63 @@ namespace Functions
     private:
       const BoundingBox<dim> bounding_box;
     };
+
+
+    /**
+     * Signed-distance level set function of a Zalesak's disk proposed in
```

Comment:
I believe this "a" should go away, i.e. "Signed-distance level set function of Zalesak's disk proposed in ...".

Otherwise I think this looks great now. :100: